rpm/systemd/diamond.service

```diff
@@ -2,8 +2,9 @@
 Description=diamond - A system statistics collector for graphite
 
 [Service]
-ExecStart=/usr/bin/python /usr/bin/diamond --log-stdout --foreground
-Restart=on-abort
+ExecStart=/usr/bin/python /usr/bin/diamond -p /var/run/diamond.pid
```

Comment:
I have modified the PR, let me know if it looks ok,  or needs change before merge/not merge